### DISCUSSION TITLE
🧩 FEATURE REQUEST: An (in-line?) comment box on the widget

### DISCUSSION THREAD

**Discussion by AnishDe12020** [dailydotdev/daily#615] [Feature Requests]
## Feature Request

An in-line (or maybe I would say on-widget) comment box instead of the modal that pops up for leaving comments

### Description

Currently, we can see the comments on the widget itself but clicking on the box to comment opens up a modal to leave the comment. Although this might be fine, it also obstructs the view of the article. The modal approach was fine on the daily.dev app but on the widget, one might want to be scrolling through the article (probably copy-pasting something?) and writing the comment (also useful for cross-checking blah blah). 

Obviously, the one advantage to the current approach is the shared modal component but then probably the body part of the modal (that is the raw comment box + preview + submit button) can be a shared component, and then it can be used for the modal as well as in the widget.

### Solution
Moving the comment writing box to the widget from the middle

**Comment by idoshamun** [comment]
Hey @AnishDe12020,

This is a great suggestion. We will consider changing this behavior once we start working on v2 of the companion
